.kitchen.yml

```diff
@@ -18,5 +18,5 @@ suites:
       - 'looker'
     attributes:
       looker:
-        startup_script_url: ENV['LOOKER_STARTUP_SCRIPT_URL']
-        jar_file_url: ENV['LOOKER_JAR_URL']
+        startup_script_url: <%= ENV['LOOKER_STARTUP_SCRIPT_URL'] %>
```

Comment:
Ok, might be worth putting into the README about how to get them.

Rest of it LGTM